addons/nightvision/XEH_postInit.sqf

```diff
@@ -43,6 +34,12 @@ GVAR(ppeffectBlur) = -1;
     ["ace_firedPlayer", LINKFUNC(onFiredPlayer)] call CBA_fnc_addEventHandler;
```

Comment:
There's a check at the beginning of the event handler. Would you prefer to remove and add the event handlers dynamically?

Reply:
`fired` event is very critical. Everyone hates fps drops when firing. So we shouldn't make it worse.
Even if it's just a tiny if/exitWith check.

Reply:
Yeah, let's change this.

Reply:
now sets firedEH to only run when setting is enabled AND nvg vision is actually active 
(it is more calls to CBA_fnc_removeEventHandler and CBA event IDs are unique, but I don't think it will be a problem)